superset/models.py

```diff
@@ -1661,7 +1661,11 @@ def get_druid_version(self):
         endpoint = (
             "http://{obj.coordinator_host}:{obj.coordinator_port}/status"
         ).format(obj=self)
-        return json.loads(requests.get(endpoint).text)['version']
+        ver = json.loads(requests.get(endpoint).text)['version']
```

Comment:
I'd prefer an approach where we can specify the DRUID_VERSION as a configuration element. I would have to be associated with the DruidCluster model. Here we'd use the information defined manually and fallback on this current approach when not specified.